Software/Python/line_follower/line_sensor.py

```diff
@@ -195,6 +203,7 @@ def line_position():
 	curr_pos=0
 	percent_black_line=[0]*5
 	for i in range(5):
-		percent_black_line[i]=diff_val[i]*100/range_col[i]
+		# casting to int when moving to Python 3
+		percent_black_line[i]=(diff_val[i]*100/range_col[i])
```

Comment:
@karan, I ended up not putting the cast to int here but please take a look. It may be needed.